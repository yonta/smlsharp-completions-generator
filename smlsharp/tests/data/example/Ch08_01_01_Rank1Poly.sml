fun f x y = (x,y);

f 1;

it "ML";

