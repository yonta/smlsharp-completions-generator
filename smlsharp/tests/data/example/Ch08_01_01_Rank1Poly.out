val f = fn : ['a. 'a -> ['b. 'b -> 'a * 'b]]

val it = fn : ['a. 'a -> int * 'a]

val it = (1, "ML") : int * string
